src/structure/pdb.jl

```diff
@@ -30,26 +39,231 @@ end
 
 
 """
-Download a Protein Data Bank (PDB) file or biological assembly from the RCSB
-PDB. By default downloads the PDB file; if `ba_number` is set the biological
-assembly with that number will be downloaded.
+Returns a list of all PDB entries from RCSB PDB server
 """
-function downloadpdb(pdbid::AbstractString,
-                    out_filepath::AbstractString="$pdbid.pdb";
-                    ba_number::Integer=0)
+function getallpdbentries()
+    pdbidlist = Array{String,1}()
+    info("Fetching list of all PDB Entries from RCSB PDB Server...")
+    download("ftp://ftp.wwpdb.org/pub/pdb/derived_data/index/entries.idx","entries.idx")
+    open("entries.idx") do input
```

Comment:
Given that we make then delete the file here, I wonder if we should use a temporary filepath? If the user has `entries.idx` in their directory here it is getting overwritten without warning. You can use `tempname()` to save a name to a variable, write to this then delete it (I think there are examples of this in the PDB tests).